### PR TITLE
[OP#41675] Add a separator between workpackages in project tab

### PR DESCRIPTION
**Pull request by SwikritiT**
OP#41675: https://community.openproject.org/projects/nextcloud-integration/work_packages/41675/activity

This PR adds a separator between the work packages
seperator
single

### CODE REVIEW

**Comment by individual-it** [comment]
CI should be fixed after https://github.com/nextcloud/integration_openproject/pull/95

File: src/views/ProjectsTab.vue [hunk 1]
```diff
@@ -31,9 +31,13 @@
...
 			</div>
-			<WorkPackage v-for="workpackage in workpackages"
+			<WorkPackage v-for="(workpackage, index) in workpackages"
 				:key="workpackage.id"
-				:workpackage="workpackage" />
+				:workpackage="workpackage">
+				<template #seperator>
+					<div v-if="index !== workpackages.length-1 " class = "workpackage-seperator"/>
+				</template>
+			</WorkPackage>
```
Comment (kiranparajuli589) [inline src/views/ProjectsTab.vue:40]: can we do this just by adding class `workpackage-separator` here, on the work package component itself?
```
<WorkPackage
  ...
  :class="{ifNotLast: 'workpackage-separator'}"
/>
```
just for border-bottom, I think this should work.

**Comment by individual-it** [comment]
rebasing should fix CI

**Review by kiranparajuli589 — APPROVED** [review]

**Comment by kiranparajuli589** [comment]
js lint failed https://github.com/nextcloud/integration_openproject/runs/5766712316?check_suite_focus=true

**Pull request merged**